pynucastro/networks/rate_collection.py

```diff
@@ -381,10 +381,13 @@ def plot(self, outfile=None, rho=None, T=None, comp=None, size=(800, 600), dpi=1
 
         fig.set_size_inches(size[0]/dpi, size[1]/dpi)
 
+        if title is not None:
+            fig.suptitle(title)
+
         if outfile is None:
             plt.show()
         else:
-            plt.tight_layout()
+            #plt.tight_layout()
```

Comment:
Is it intentional to leave `plt.tight_layout()` commented out?

Reply:
@zingale did you still want this commented?

Reply:
you can revert it

Reply:
uncommented from slack discussion